src/pseuplex/app.ts

```diff
@@ -501,6 +501,78 @@ export class PseuplexApp {
 			})
 		]);
 
+		router.get('/hubs/sections/:sectionId', [
```

Comment:
To try to keep things decoupled, almost all the routes implemented in `PseuplexApp` call "response modifiers", which are functions that each plugin can define to modify the response body. In this case, it might be better if you rework this to do something similar, and then update the section response content from a plugin's response modifier instead

Reply:
A good place to see this is in the letterboxd plugin